pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -6,6 +6,9 @@
 , abseil-cpp
 , cmake
 , eigen
+, flatbuffers
```

Comment:
i'm surprised `flatbuffers` works given that it should be a clear version mismatch

Reply:
Well, see https://github.com/NixOS/nixpkgs/pull/330416